pgx/src/bgworkers.rs

```diff
@@ -9,7 +9,7 @@ Use of this source code is governed by the MIT license that can be found in the
 
 //! Safely create Postgres Background Workers, including with full SPI support
 //!
-//! See: [https://www.postgresql.org/docs/12/bgworker.html](https://www.postgresql.org/docs/12/bgworker.html)
+//! See: [https://www.postgresql.org/docs/14/bgworker.html](https://www.postgresql.org/docs/14/bgworker.html)
```

Comment:
Let's make this the last time we have to change the URL, instead:
```suggestion
//! See: [https://www.postgresql.org/docs/current/bgworker.html](https://www.postgresql.org/docs/current/bgworker.html)
```